servers/rendering/renderer_rd/forward_clustered/render_forward_clustered.h

```diff
@@ -503,6 +503,7 @@ class RenderForwardClustered : public RendererSceneRenderRD {
 				uint64_t priority : 8;
 				uint64_t geometry_id : 32;
 
+				uint64_t material_depth : 32;
```

Comment:
This overflows sort_key2 so we will lose the `shader_id` when sorting opaque objects